objects/endpoint.rb

```diff
@@ -79,7 +79,9 @@ def to_h
       log: @item['log']
     }
     h[:updated] = Time.at(@item['updated']) if @item['updated']
-    h[:flipped] = Time.at(@item['flipped']) if @item['flipped']
+    # TODO: #75 consider other implementations of the null Time object that may
+    # be better than Time.new(0).
+    h[:flipped] = @item['flipped'] ? Time.at(@item['flipped']) : Time.new(0)
```

Comment:
@pdacostaporto I don't think it's the best approach though. Instead of setting `flipped` to some dumb value we better check its presence at the presentation layer and render the page differently if it's absent. Don't you think?